packages/@aws-cdk/aws-lambda/lib/event-source-mapping.ts

```diff
@@ -101,6 +103,20 @@ export interface EventSourceMappingOptions {
    * @default - no topic
    */
   readonly kafkaTopic?: string;
+
+  /**
+   * The Secrets Manager secret ARN that stores your broker credentials.
+   *
+   * @default - no configuration
+   */
+  readonly kafkaSecretArn?: string
+
+  /**
+   * A list of Kafka bootstrap servers to connect to your self managed Kafka cluster
+   *
+   * @default - none
+   */
+  readonly kafkaBootstrapServers?: string[]
```

Comment:
Not sure I understand what this is. Are these ARNs, URLs, endpoints or something else?

Reply:
These are endpoints (IP:PORT pairs). The name is from [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-eventsourcemapping-endpoints.html#cfn-lambda-eventsourcemapping-endpoints-kafkabootstrapservers). My assumption is, that people who want to connect to a self hosted Kafka kind of know what they are doing. The term itself is directly from the [Kafka internals](https://jaceklaskowski.gitbooks.io/apache-kafka/content/kafka-properties-bootstrap-servers.html).

Reply:
I would combine some of these documentations, to something like -

```suggestion
   * The list of host and port pairs that are the addresses of the Kafka brokers in a "bootstrap" Kafka cluster that
   * a Kafka client connects to initially to bootstrap itself. They are in the format `abc.xyz.com:xxxx`.
   *
   * @default - none
   */
  readonly kafkaBootstrapServers?: string[]
```